_posts/2019-02-25-the-western-grid.md

```diff
@@ -0,0 +1,136 @@
+    <img src="{{ "/images/SaltLakeHistoric_sm.jpg" | prepend: site.baseurl }}" alt="1891 Salt Lake City Bird's Eye View">
+  </a>
+  <figcaption class="caption__text">Fig 1: 1891 Salt Lake City Bird's Eye View</figcaption>
+</figure>
+
+Have you ever wondered why the patterns of development in the Western United States are so orthogonal (i.e., "right-angled")? Pan around a map and you'll see the confines of the grid just about everywhere. Grid-like development is nothing new - it can be traced back to early developments in the Middle East. But, why is it so prevalent in the West?
+
+It turns out this gridiron pattern is connected to a 17th-century mathematician, the Treaty of Paris, and Thomas Jefferson. Yup, there's definitely a connection here, which also helps explain the West's notoriously wide streets and big city blocks, as well as our nation’s entangled dependence on feet, yards, and miles (i.e., the United States customary units). Read on and we will explore the Western Grid more.
+
+## The Survey Chain
+{: .text-left}
+
+<figure class="caption caption--right">
+  <img src="{{ "/images/GunterChain.png" | prepend: site.baseurl }}" alt="Gunter Chain" />
+  <figcaption class="caption__text">Fig 2: Gunter Chain</figcaption>
+</figure>
+
+Let’s start with the previously mentioned 17th-century mathematician: Edmund Gunter. Gunter was not just a mathematician; he was also an English clergyman, geometer, and astronomer. He made several contributions to society, but we’re going to focus on his invention of the Gunter Chain in 1620 (Fig 2).
+The Gunter Chain, which was used for surveying, was quickly adopted as a statutory measurement in England and in the British Empire. What made it so popular was its ease of measuring and surveying plots of land for legal purposes. The Gunter Chain measures 66 feet in length and contains 100 links, making it very portable and convenient when working with English Units. Measuring an acre (a standard British parcel) was as easy as laying out 10 square chains. When working with smaller plots of land, you'd simply divide the links by 100,000 to get the acres. Additionally, a statute mile was 80 chains.
+Because of its ease in calculations, the Gunter Chain was quickly adopted and became widely used for surveying and land management records. We’ll see its lasting effect as we begin to look closer at the Western United States grid.
+
+The Gunter Chain made its way across the Atlantic with the early colonists. It was so popular that the acre, noted in the Articles of Confederation (1781), was declared the standard unit of land measurement for the new nation.
+
+## Standard Units of Measure (The Acre)
+{: .text-left}
+
+<figure class="caption caption--right">
+  <img src="{{ "/images/NorthwestTerritory.png" | prepend: site.baseurl }}" alt="Northwest Territory" />
+  <figcaption class="caption__text">Fig 3: Northwest Territory</figcaption>
```

Comment:
Footnote 3: 

Golbez, “States and Territories of the United States of America August 7 1789 to April 2 1790,” Wikipedia, “Northwest Territory” page, accessed March 1, 2019. https://en.wikipedia.org/wiki/Northwest_Territory.